include/rogue/EnableSharedFromThis.h

```diff
@@ -28,13 +25,13 @@
 namespace rogue {
 
 class EnableSharedFromThisBase : public std::enable_shared_from_this<rogue::EnableSharedFromThisBase> {
-  public:
+ public:
     virtual ~EnableSharedFromThisBase() {}
 };
 
 template <typename T>
 class EnableSharedFromThis : virtual public EnableSharedFromThisBase {
-  public:
+ public:
```

Comment:
Did the linter suggest this spacing?
The previous spacing was set by clang-format.

Reply:
https://github.com/slaclab/rogue/commit/cf1635ea5edf421d12c48113246818289e34e362

Reply:
```
resolving c++ linter
public: should be indented +1 space inside class [whitespace/indent]
```

Reply:
https://guiquanz.gitbooks.io/google-cc-style-guide/content/ebook/Formatting.html
![image](https://github.com/slaclab/rogue/assets/11821758/4f34d3a5-fc8a-48bf-92a9-838727a4acd5)

Reply:
[reran clang-format to sync up with cpplint](https://github.com/slaclab/rogue/pull/1012/commits/bbee74378a5052505e45828a7d0823058c54ec7f)